src/ast/mod.rs

```diff
@@ -1271,6 +1271,11 @@ pub enum Statement {
         variable: ObjectName,
         value: Vec<Expr>,
     },
+    /// SET TIME ZONE <value>
+    ///
+    /// Note: this is a PostgreSQL-specific statements
+    /// SET TIME ZONE <value> is an alias for SET timezone TO <value> in PostgreSQL
+    SetTimeZone { local: bool, value: Expr },
```

Comment:
add a new Statement SetTimeZone to reflect the difference between `SET TIME ZONE TO 'UTC'` and `SET TIME ZONE 'UTC'`